modules/ding_user/ding_user.module

```diff
@@ -983,87 +808,51 @@ function ding_user_forms($form_id, $args) {
 }
 
 /**
- * Ajax command to authenticate. Used by ding_provider.
+ * Ajax command to authenticate.
+ *
+ * @param mixed $extra_data
+ *   Options to send to ding_popup.
+ * @param string $redirect_uri
+ *   Destination to redirect to after login for ding_adgangsplatformen.
```

Comment:
This refers to `ding_adgangsplatformen`. That does not seem right if this module does not depend on `ding_adgangsplatformen`?

Reply:
Fixed